configs/envoy-demo.yaml

```diff
@@ -40,6 +40,7 @@ static_resources:
   clusters:
   - name: service_envoyproxy_io
     type: LOGICAL_DNS
+    connect_timeout: 10s
```

Comment:
@Cosmicoppai this isnt necessary (and hasnt been for a long time)

i think the solution here is to remove the deb instructions as they reference an old version of envoy which is unlikely to work with current docs